packages/core/admin/ee/admin/pages/SettingsPage/pages/AuditLogs/ListView/ComboboxFilter/index.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Combobox, ComboboxOption } from '@strapi/design-system/Combobox';
+import { createGlobalStyle } from 'styled-components';
+
+const GlobalOverflowAuto = createGlobalStyle`
+  /* Prevent scroll on combobox click when the page has an overflow */
+  html, body {
+    overflow: auto;
+  }
+`;
+
+const ComboboxFilter = ({ value, options, onChange }) => {
+  return (
+    <>
+      <GlobalOverflowAuto />
+      <Combobox aria-label="combobox input" value={value} onChange={onChange}>
```

Comment:
This label would need to be translated and doesn't make much sense to a user, don't you think?

Reply:
Thanks, I did notice though that [the Combobox ](https://github.com/strapi/design-system/blob/62c8e8d40a9c94ab2bd9f5d0aaea597c534491bb/packages/strapi-design-system/src/Combobox/Combobox.js#L299-L305) doesn't appear to be using the aria-label. It throws an error but I don't see where it is passed to the input. Am I missing something or should I open a PR in the DS?

Reply:
Oh yeah. @joshuaellis is currently refactoring the `Select` and `Combobox`. Maybe it is good to take that into account?

Reply:
With the right DOM it shouldn't need the aria-label attribute because the combobox should have text / label, i'll make sure to check these things when I work on it 👍🏼